entity-service/build.gradle.kts

```diff
@@ -14,13 +14,15 @@ tasks.integrationTest {
 
 dependencies {
   implementation(project(":entity-service-impl"))
+  implementation(project(":entity-service-change-event-impl"))
```

Comment:
naming - it does use the new api but I think something like `:entity-service-change-event-generator` would be clearer.

Reply:
done